src/profile-logic/symbol-store-db.js

```diff
@@ -38,7 +40,7 @@ const kTwoWeeksInMilliseconds = 2 * 7 * 24 * 60 * 60 * 1000;
  * @class SymbolStoreDB
  * @classdesc Where does this description show up?
  */
-export class SymbolStoreDB {
+export default class SymbolStoreDB {
```

Comment:
This is done so that I could mock it with jest... I couldn't find how to do with a named export :/